test/ceph-fio/README

```diff
@@ -0,0 +1,7 @@
+Document explain:
+
+fio-init.conf: Before test, exec"fio fio-init.conf" to write 8 images.
+fio-env.conf: It's a fio conf, using env parameter to decide FIO_SIZE, FIO_DEPTH, FIO_JOBS.
```

Comment:
Please rename the file to fio-env.conf instead of fio-env.sh

Reply:
done